### PR TITLE
Links in markdown

### PR DESCRIPTION
**Pull request by chrisnojima**
Creates clickable links. Regexp isn't perfect but its fuzzy enough i think. Better than what we have now

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 955198c1604b...0936c332c2f2 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Markdown-links**  
   Markdown-links rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 955198c1604b...6ec5c8a2dc44 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Markdown-links**  
   Markdown-links rendered
</summary>

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by keybase-ci-visdiff** [comment]
The commits 955198c1604b...40b15650a94e introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Markdown-links**  
   Markdown-links rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 955198c1604b...de4d9ae0a5eb introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Markdown-links**  
   Markdown-links rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 955198c1604b...f935fe98daa9 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Markdown-links**  
   Markdown-links rendered
</summary>

File: shared/common-adapters/markdown.js [hunk 1]
```diff
@@ -63,12 +65,21 @@ function previewCreateComponent (type, key, children) {
   }
 }
 
+function onClickLink (e) {
+  if (e.target.innerText) {
+    // $FlowIssue doens't understand innerText
+    openURL(e.target.innerText)
```
Comment (MarcoPolo) [inline shared/common-adapters/markdown.js]: Is this the same on native?

Comment (chrisnojima) [inline shared/common-adapters/markdown.js]: doubt it, we'll likely have to split this up

**Pull request merged**